service/src/main/kotlin/fi/espoo/evaka/shared/dev/DataInitializers.kt

```diff
@@ -533,6 +534,15 @@ RETURNING id
     """.trimIndent()
 )
 
+fun Database.Transaction.insertTestPricing(pricing: FeeThresholdsWithValidity) = insertTestDataRow(
```

Comment:
I think it would be a good idea to get rid of the old "pricing" name.

Reply:
Let's do this exhaustively later